client/plots/hierCluster.config.js

```diff
@@ -18,7 +18,7 @@ export async function getPlotConfig(opts = {}, app) {
 		- non-gene genomic stuff that resolves into numeric quantities (cpg meth)
 		- metabolite
 		*/
-		dataType: dtgeneexpression,
+		dataType: opts.termgroups?.[0]?.termType || TermTypes.GENE_EXPRESSION,
```

Comment:
code that's launch hiercluster should be required to specify datatype. if missing should abort, and not to default to gene exp